providers/implementations/rands/seeding/rand_unix.c

```diff
@@ -33,6 +33,9 @@
 # include <sys/sysctl.h>
 # include <sys/param.h>
 #endif
+#if defined(__FreeBSD__)
+#include <sys/random.h>
```

Comment:
Since what version does that exist? Do we need to add something like  __FreeBSD_version >= 1200061?

Reply:
<sys/random.h> has been there since 1995 and predates even the sysctl that openssl uses without a version check, but I suppose we could scope this include to the versions we'll use getrandom(2) for.